From: Antoine Musso <hashar@free.fr>
Date: Tue, 10 Mar 2015 12:48:13 +0100
Subject: Replace python shebang with python2.7

The Precise package is only meant for python2.7, get rid of some lintian
errors by pointing to python2.7.

Change-Id: I68b805d0e1b49971063ed98ccc187f4456f9501d
---
 setup.py                      | 2 +-
 tests/base.py                 | 2 +-
 tests/test_cloner.py          | 2 +-
 tests/test_layoutvalidator.py | 2 +-
 tests/test_merger_repo.py     | 2 +-
 tests/test_requirements.py    | 2 +-
 tests/test_scheduler.py       | 2 +-
 tests/test_webapp.py          | 2 +-
 tests/test_zuultrigger.py     | 2 +-
 tools/trigger-job.py          | 2 +-
 tools/zuul-changes.py         | 2 +-
 zuul/cmd/__init__.py          | 2 +-
 zuul/cmd/client.py            | 2 +-
 zuul/cmd/cloner.py            | 2 +-
 zuul/cmd/merger.py            | 2 +-
 zuul/cmd/server.py            | 2 +-
 16 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/setup.py b/setup.py
index 70c2b3f..4ec20a6 100644
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 # Copyright (c) 2013 Hewlett-Packard Development Company, L.P.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
diff --git a/tests/base.py b/tests/base.py
index 38d2817..6d4f017 100755
--- a/tests/base.py
+++ b/tests/base.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 
 # Copyright 2012 Hewlett-Packard Development Company, L.P.
 #
diff --git a/tests/test_cloner.py b/tests/test_cloner.py
index e3576bd..064a61c 100644
--- a/tests/test_cloner.py
+++ b/tests/test_cloner.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 
 # Copyright 2012 Hewlett-Packard Development Company, L.P.
 # Copyright 2014 Wikimedia Foundation Inc.
diff --git a/tests/test_layoutvalidator.py b/tests/test_layoutvalidator.py
index 46a8c7c..101e0d0 100644
--- a/tests/test_layoutvalidator.py
+++ b/tests/test_layoutvalidator.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 
 # Copyright 2013 OpenStack Foundation
 #
diff --git a/tests/test_merger_repo.py b/tests/test_merger_repo.py
index 454f3cc..55d16a0 100644
--- a/tests/test_merger_repo.py
+++ b/tests/test_merger_repo.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 
 # Copyright 2012 Hewlett-Packard Development Company, L.P.
 # Copyright 2014 Wikimedia Foundation Inc.
diff --git a/tests/test_requirements.py b/tests/test_requirements.py
index 3ae56ad..202010c 100644
--- a/tests/test_requirements.py
+++ b/tests/test_requirements.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 
 # Copyright 2012-2014 Hewlett-Packard Development Company, L.P.
 #
diff --git a/tests/test_scheduler.py b/tests/test_scheduler.py
index 628775d..ac70091 100755
--- a/tests/test_scheduler.py
+++ b/tests/test_scheduler.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 
 # Copyright 2012 Hewlett-Packard Development Company, L.P.
 #
diff --git a/tests/test_webapp.py b/tests/test_webapp.py
index 94f097a..93d88a8 100644
--- a/tests/test_webapp.py
+++ b/tests/test_webapp.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 
 # Copyright 2014 Hewlett-Packard Development Company, L.P.
 # Copyright 2014 Rackspace Australia
diff --git a/tests/test_zuultrigger.py b/tests/test_zuultrigger.py
index 0d52fc9..49b79ad 100644
--- a/tests/test_zuultrigger.py
+++ b/tests/test_zuultrigger.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 
 # Copyright 2014 Hewlett-Packard Development Company, L.P.
 #
diff --git a/tools/trigger-job.py b/tools/trigger-job.py
index 7123afc..0c307ca 100755
--- a/tools/trigger-job.py
+++ b/tools/trigger-job.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 # Copyright 2013 OpenStack Foundation
 #
 # Licensed under the Apache License, Version 2.0 (the "License"); you may
diff --git a/tools/zuul-changes.py b/tools/zuul-changes.py
index 8b854c7..e156268 100755
--- a/tools/zuul-changes.py
+++ b/tools/zuul-changes.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 # Copyright 2013 OpenStack Foundation
 # Copyright 2015 Hewlett-Packard Development Company, L.P.
 #
diff --git a/zuul/cmd/__init__.py b/zuul/cmd/__init__.py
index 5ffd431..120c62c 100644
--- a/zuul/cmd/__init__.py
+++ b/zuul/cmd/__init__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 # Copyright 2012 Hewlett-Packard Development Company, L.P.
 # Copyright 2013 OpenStack Foundation
 #
diff --git a/zuul/cmd/client.py b/zuul/cmd/client.py
index 1ce2828..61183da 100644
--- a/zuul/cmd/client.py
+++ b/zuul/cmd/client.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 # Copyright 2012 Hewlett-Packard Development Company, L.P.
 # Copyright 2013 OpenStack Foundation
 #
diff --git a/zuul/cmd/cloner.py b/zuul/cmd/cloner.py
index 4f8b9f4..dbbe396 100755
--- a/zuul/cmd/cloner.py
+++ b/zuul/cmd/cloner.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 #
 # Copyright 2014 Antoine "hashar" Musso
 # Copyright 2014 Wikimedia Foundation Inc.
diff --git a/zuul/cmd/merger.py b/zuul/cmd/merger.py
index 797a990..2807f9e 100644
--- a/zuul/cmd/merger.py
+++ b/zuul/cmd/merger.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 # Copyright 2012 Hewlett-Packard Development Company, L.P.
 # Copyright 2013-2014 OpenStack Foundation
 #
diff --git a/zuul/cmd/server.py b/zuul/cmd/server.py
index 0b7538d..59834ad 100755
--- a/zuul/cmd/server.py
+++ b/zuul/cmd/server.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 # Copyright 2012 Hewlett-Packard Development Company, L.P.
 # Copyright 2013 OpenStack Foundation
 #
